include/odp/api/spec/packet_io.h

```diff
@@ -489,6 +489,8 @@ typedef union odp_pktout_config_opt_t {
 		 */
 		uint64_t no_packet_refs  : 1;
 
+		/** Enable packet Tx completion event */
+		uint64_t tx_compl_ena : 1;
```

Comment:
I think some body text is needed to describe that this flag alone does not enable event sending. So, something like this could be added:

/** Enable packet transmit completion event requests
   *
   * Use pktio capability tx_compl to check if TX completion events are supported.
   * 
   * 0: Application will not request packet TX completion events (default)
   * 1: Application will request packet TX completion events
*/